prediction_market_agent/agents/microchain_agent/prompt_handler.py

```diff
@@ -0,0 +1,34 @@
+from loguru import logger
+from prediction_market_agent_tooling.tools.utils import utcnow
+
+from prediction_market_agent.db.db_storage import DBStorage
+from prediction_market_agent.db.models import Prompt
+
+
+# ToDo - Unify PromptHandler, db_storage and LongTermMemory into 2 classes, one per table.
```

Comment:
Do you want to do that as part of this PR or a follow-up one? If follow-up, could you make an issue and tag it here?

Reply:
There is a separate issue for that, see https://github.com/gnosis/prediction-market-agent/issues/242

Reply:
Could you link that PR here so it doesn't get lost?